libr/core/gdiff.c

```diff
@@ -159,3 +159,79 @@ R_API void r_core_diff_show(RCore *c, RCore *c2) {
 		}
 	}
 }
+
+/* Iterate available diffs and print json output */
+R_API void r_core_diff_show_json(RCore *c, RCore *c2) {
+	RList *fcns = r_anal_get_fcns (c->anal);
+	const char *match;
+	RListIter *iter;
+	RAnalFunction *f;
+	PJ *pj = r_core_pj_new (c);
+
+	if (!pj) {
+		return;
+	}
+
+	fcns = r_anal_get_fcns (c->anal);
```

Comment:
This var is assigned twice

Reply:
The calls are different, first one's argument is `c->anal` while the second one is `c2->anal`.